docs/plugins.md

```diff
@@ -3,48 +3,49 @@
 Is Snapcraft missing support for your preferred build-system? Here's how you
 would add it!
 
-Snapcraft can be extended by adding new part plugins. Let's add support for
-running a custom build-tool named "Crafty". The goal of this plugin is to
-pass a configurable target to the `crafty sometarget` command inside the
-`src/crafty` source directory.
+Snapcraft can be extended by adding new part plugins, which are written in
+Python. Let's add support for running a custom build-tool named "Crafty". The
+goal of this plugin is to pass a configurable target to the `crafty sometarget`
+command inside the `src/crafty` source directory.
 
-Relative to the directory of `snapcraft.yaml`, create 
-`parts/plugins/x-crafty.yaml` with:
-
-```yaml
-options:
-    source:
-        required: true
-    source-type:
-    source-tag:
-    source-branch:
-    crafty-target:
-        required: true
-```
-
-and `parts/plugins/x_crafty.py` with:
+Relative to the directory of `snapcraft.yaml`, create
+`parts/plugins/crafty.py` with:
```

Comment:
That just overrides any plugin named "crafty"' shipped in Snapcraft, right? I figured it would be best to simply discuss how plugins are written and then mention (lines 38-39) that the x_ would make it take precedence over in-Snapcraft plugins.

By the way, is there any reason not to simply take local plugins over Snapcraft plugins and get rid of the whole x_ thing?